build/repo/gov/nasa/jpl/oodt/oodt-core/2.0/oodt-core-2.0.pom

```diff
@@ -28,7 +28,7 @@ $Id$
     <system>JIRA</system>
     <url>https://osr.jpl.nasa.gov/jira/secure/ViewProject.jspa?pid=10000</url>
   </issueManagement>
-  <repositories>
+  <!--repositories>
```

Comment:
Rather than comment-out lines, just nuke 'em! (If you really want to keep them, then add a comment explaining why.)